docs/content/specs-defs/specs/terraform/resource/interfaces.md

```diff
@@ -125,25 +125,11 @@ In Terraform, locks become part of the resource graph and suitable `depends_on`
 ## Private Endpoints
 
 {{< tabs "private-endpoints" >}}
-  {{< tab "Bicep User Defined Type, Parameter & Resource Example" >}}
-  {{< hint type=note >}}
-  Please note that you will need to ensure that the User-Defined Types for [Role Assignments](#role-assignments) & [Locks](#resource-locks) also are present in your module file for this interface to work correctly.
-  {{< /hint >}}
-  {{< expand "➕ Variant 1: A default service (`groupId`) can be assumed - e.g., for services that only have one private endpoint type)" "expand/collapse" >}}
-  {{< include file="/static/includes/interfaces/int.pe.udt.schema1.bicep" language="bicep" options="linenos=false" >}}
-  {{< /expand >}}
-  {{< expand "➕ Variant 2: A default service (`groupId`) cannot be assumed - e.g., for services that have more than one private endpoint type, like a Storage Account (blob, file, etc.)" "expand/collapse" >}}
-  {{< include file="/static/includes/interfaces/int.pe.udt.schema2.bicep" language="bicep" options="linenos=false" >}}
-  {{< /expand >}}
-  {{< /tab >}}
-  {{< tab "Bicep Input Example with Values" >}}
-  {{< include file="/static/includes/interfaces/int.pe.input.bicep" language="bicep" options="linenos=false" >}}
-  {{< /tab >}}
```

Comment:
Differently than others just moved, this content looks removed. Just double checking if that's intended

Reply:
That was intentional and a mistake by the original auther. If you look closely, it references Bicep code, even though it is the TF. The correct TF code is referenced below